hibike/sensor_testing/sensor_testing/templates/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="charset=utf-8" />
+        <title>Sensor Testing Web Interface</title>
+        <style>
```

Comment:
This CSS in HTML files is okay for now but if we end up adding more settings, we should consider moving this to its own file for clarity